scripts/scheming_migration/index.js

```diff
@@ -148,10 +148,22 @@ async function main() {
 			for (const key in contactsObj) {
 				packageExtras['contacts'].push(contactsObj[key]);
 			}
+
 			packageExtras['dates'] = [];
+			let hasCreatedDate = false;
+
 			for (const key in datesObj) {
 				packageExtras['dates'].push(datesObj[key]);
+
+				if (datesObj[key]['type'] === 'Created') {
+					hasCreatedDate = true;
+				}
+			}
+
+			if (!hasCreatedDate && (resourceType == 'webservice' ||  resourceType ===  'application')) {
+				packageExtras['dates'].push({"type": "Created", "date": packageExtras['record_create_date']});
```

Comment:
A comment to explain why this is being done and especially why it's specifically targeting webservices and applications would be helpful.

Reply:
> The 'dates' field has two primary options, Published and Created and is a mandatory field. Almost all other resources in the DB have ether a Published or a Created date associated with them (there may be specific cases where they dont, but those would be in error and the script is not meant to handle them). But 'webservice' and 'application' never had a Published or Created date associated with them in this format and since we are making it a mandatory field, instead of breaking all the 'webservice' and 'application' packages, it was decided to add a Created date, based on their record_create_date.

I see. I'm still leaning towards handling the general case of no Created and no Published, but if you feel it's out of scope fair enough. I'm marking this one resolved.